fuel-txpool/src/containers/dependency.rs

```diff
@@ -246,6 +247,8 @@ impl Dependency {
                                 .expect("Tx should be always present in txpool");
                             // compare if tx has better price
                             if txpool_tx.gas_price() > tx.gas_price() {
+                                // TODO: should we consider the value of all dependant txs to avoid
+                                //       large graphs being cancellable by single tx with a slightly higher gas price?
```

Comment:
ah good point, I was thinking more in terms of someone griefing a bunch of queue txs on a contract creation output, since anyone could observe and then make the same contract id output. However, that's a totally different path than this part.